.sync.yml

```diff
@@ -2,7 +2,9 @@
 .travis.yml:
```

Comment:
this file isn't used anymore and was forgotten in the last modulesync. can you delete it please?